docs/logging-and-managing-experiment-results/index.rst

```diff
@@ -2,101 +2,111 @@
 
 Logging and managing experiment results
 =======================================
+What is neptune-client?
+-----------------------
+|neptune-client-github| is an open source Python library that serves two purposes:
 
-|Youtube Video|
+1. logging machine learning experiments,
+2. fetching experiment data from Neptune to local machine.
 
-|neptune-client-github|
-is an open source Python library that lets you integrate your Python scripts with Neptune so that you can more easily track and organize your experiments in the rich Neptune |dashboard|.
+It is designed to be **lightweight** (fast onboarding), **generic** (capable of logging any kind of machine learning work) and **straightforward** (user defines what to keep track of during experiment) to use.
```

Comment:
fast onboarding is more and information for us than for the user -> I'd go with low overhead or low setup effort.

Reply:
thanks, corrected.